### PR TITLE
AA-448: UI fixes

### PR DESCRIPTION
**Pull request by Dillon-Dumesnil**
LinkedIn button size and single column for view grades button in mobile

### CODE REVIEW

**Comment by Dillon-Dumesnil** [comment]
LinkedIn Button resizing: 
image
Two column view for view grades: 
image
Mobile view for view grades (single column): 
image

**Review by mikix — APPROVED** [review]
:bless:

**Review by ekangedx — APPROVED** [review]

File: src/courseware/course/course-exit/CourseNonPassing.jsx [hunk 1]
```diff
@@ -33,12 +33,12 @@ function CourseNonPassing({ intl }) {
         <div className="col-12 p-0 h2 text-center">
           { intl.formatMessage(messages.endOfCourseHeader) }
         </div>
-        <Alert variant="primary" className="col col-lg-10 mt-4 d-flex align-items-start">
-          <div className="flex-grow-1 mr-5">{ intl.formatMessage(messages.endOfCourseDescription) }</div>
+        <Alert variant="primary" className="row col col-lg-10 mt-4 d-flex align-items-start">
```
Comment (ciduarte) [inline src/courseware/course/course-exit/CourseNonPassing.jsx]: I don't think it's best practice to use both row and col in the same element. I think you should leave `<Alert />` as the column, and wrap it's children in a row div..

**Comment by ekangedx** [comment]
Looks awesome @Dillon-Dumesnil, thanks!!!

**Review by ciduarte — APPROVED** [review]
I'll take it

**Pull request merged**